spec/services/subject_mapper_spec.rb

```diff
@@ -136,7 +73,7 @@
       },
       {
         course_title: "",
-        ucas_subjects: %w[secondary welsh],
+        ucas_subjects: %w[welsh],
```

Comment:
```suggestion
ucas_subjects: %w[secondary welsh],
```

put that back in as its a port from csharp

Reply:
Welsh logic completely excised in f7b1d63 (it's no longer needed)